pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -465,12 +469,12 @@ func (tc *NoExecuteTaintManager) handleNodeUpdate(ctx context.Context, nodeUpdat
 	}
 
 	// Create or Update
-	klog.V(4).InfoS("Noticed node update", "node", nodeUpdate)
+	logger.V(4).Info("Noticed node update", "node", nodeUpdate)
 	taints := getNoExecuteTaints(node.Spec.Taints)
 	func() {
 		tc.taintedNodesLock.Lock()
 		defer tc.taintedNodesLock.Unlock()
-		klog.V(4).InfoS("Updating known taints on node", "node", node.Name, "taints", taints)
+		logger.V(4).Info("Updating known taints on node", "node", node.Name, "taints", taints)
```

Comment:
Log "node" as object. Please check the other usages of "node", some of the following ones also need to be converted.

Reply:
done